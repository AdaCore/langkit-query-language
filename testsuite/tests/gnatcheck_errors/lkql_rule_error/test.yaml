driver: gnatcheck
rules_dirs:
  - rules
input_sources:
  - main.adb
rules:
  - +RTest
