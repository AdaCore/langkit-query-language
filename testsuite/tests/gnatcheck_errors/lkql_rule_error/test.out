gnatcheck: error: test.lkql:3:05: Unknown symbol: blah
gnatcheck: error: Errors during analysis
1. Summary

   fully compliant sources               : 1
   sources with exempted violations only : 0
   sources with non-exempted violations  : 0
   unverified sources                    : 0
   total sources                         : 1
   ignored sources                       : 0

   non-exempted violations               : 0
   rule exemption warnings               : 0
   compilation errors                    : 0
   exempted violations                   : 0
   internal errors                       : 2

2. Exempted Coding Standard Violations

   no exempted violations detected

3. Non-exempted Coding Standard Violations

   no non-exempted violations detected

4. Rule exemption problems

   no rule exemption problems detected

5. Language violations

   no language violations detected

6. Gnatcheck internal errors


>>>program returned status code 2
